ChallengeTestRunner/challenge_test_lib/challenge_test.py

```diff
@@ -265,11 +265,15 @@ def challenge_fun(self, *args, **kwargs):
     def assert_printed(self, value, index=None):
```

Comment:
There should be a `msg` argument (default=None). If set it should replace the `msg` argument in the asserts.

Reply:
Done